quickjs.c

```diff
@@ -51816,6 +51816,35 @@ void JS_AddPerformance(JSContext *ctx)
     JS_FreeValue(ctx, performance);
 }
 
+/* Equality comparisons and sameness */
+JS_BOOL JS_IsLooselyEqual(JSContext *ctx, JSValue op1, JSValue op2)
+{
+    JSValue* sp = js_malloc(ctx, sizeof(JSValue) * 3);
```

Comment:
I think you should be able to do this by allocating the array on the stack.

Reply:
Thanks for your review suggestions.
I have completed the modifications in the new commit.